add-ons/istio/README.adoc

```diff
@@ -25,8 +25,9 @@ $ minishift config set memory 8GB <2>
 $ minishift config set cpus 4 <3>
 $ minishift config set image-caching true <4>
 $ minishift config set openshift-version v3.1.0 <5>
+$ minishift addon enable admin-user
+$ minishift addon enable anyuid
 $ minishift addon enable dynamic-admission-controllers #optional <6>
```

Comment:
should this go away too ?

Reply:
no that setting is required for the profile in general.